src/Blocks/QueryLoopExtension.php

```diff
@@ -43,13 +43,26 @@ public static function registerFrontendQuery(): void
             'query_loop_block_query_vars',
             function ($query, $block) {
                 $blockQuery = $block->context['query'] ?? [];
+
+                // Sanitize query only applied to ActionsList block.
+                if ((bool) array_key_exists('isCustomQueryLoop', $blockQuery)) {
```

Comment:
Need to check this to avoid apply this changes to the native `core/query` query.